src/components/templates/TemplateSearchPortal.vue

```diff
@@ -162,11 +162,13 @@
                         <p v-if="props.row.Agent !== 'NA' || isAdmin">
                             Mobile:
                             <button
+                                v-if="props.row.Agent !== 'NA'"
```

Comment:
this means if the lead is not assigned number will be visible, right?
this is not expected.
it should be like
```
<strong v-if="isAdmin">  {{ props.row.Mobile }}</strong>
<button v-else>...
```